.github/workflows/on-pr-to-main.yml

```
@@ -0,0 +1,39 @@
+name: Test Notebooks on PR to Main
+
+on:
+  pull_request:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)